src/library/scala/convert/Wrappers.scala

```diff
@@ -0,0 +1,748 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+// GENERATED CODE: DO NOT EDIT.
+
+
+package scala.convert
+
+object Wrappers {
+  private[convert] class FromJavaBiConsumer[T, U](jf: java.util.function.BiConsumer[T, U]) extends scala.Function2[T, U, Unit] {
```

Comment:
Private types shouldn't leak into public APIs, this will probably fail with an error in dotty. All these classes need to be public.

Reply:
OK this one actually doesn't, but others like `RichBiConsumerAsFunction2` do.